pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/FunctionsStatsGenerator.java

```diff
@@ -0,0 +1,78 @@
+package org.apache.pulsar.broker.stats.prometheus;
```

Comment:
Does this file belong here?

Reply:
Well the other metrics generators are here.  The generator doesn't collect metrics its just puts them into prometheus format